pkg/registry/apis/search/register.go

```diff
@@ -73,17 +81,55 @@ func (b *SearchAPIBuilder) GetAPIRoutes() *builder.APIRoutes {
 						},
 					},
 				},
-				Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-					urlQuery := r.URL.Query().Get("query")
-					searchRequest := &resource.SearchRequest{Query: urlQuery}
+				Handler: func(w http.ResponseWriter, r *http.Request) {
+					// get tenant
+					orgId, err := request2.OrgIDForList(r.Context())
+					if err != nil {
+						panic(err)
+					}
+					tenant := b.namespacer(orgId)
+
+					queryParams, err := url.ParseQuery(r.URL.RawQuery)
+					if err != nil {
+						panic(err)
```

Comment:
i think we should set the statuscode on the writer (at least) here and return rather than panic